javascript/javascript1/week2/Ali-Nematollahi-homework/getFullname.js

```diff
@@ -0,0 +1,21 @@
+console.log("====== Flight booking fullname function ======");
+
+function getFullname(firstname , surname , useFormalName , gender){
+    if(useFormalName){
+        if(gender === "male"){
```

Comment:
What if someone called the function by setting `gender` to capital first letter? It's good to use `toLowerCase()` here then.
```suggestion
        if(gender.toLowerCase() === "male"){
```